files/zh-cn/web/javascript/reference/operators/async_function/index.html

```diff
@@ -8,7 +8,7 @@
   - 实验性内容
   - 操作符
 translation_of: Web/JavaScript/Reference/Operators/async_function
-original_slug: Web/JavaScript/Reference/Operators/async允许声明一个函数为一个包含异步操作的函数
+original_slug: Web/JavaScript/Reference/Operators/async 允许声明一个函数为一个包含异步操作的函数
```

Comment:
should not be modified, please consider how to deal with it

```suggestion
original_slug: Web/JavaScript/Reference/Operators/async允许声明一个函数为一个包含异步操作的函数
```

Reply:
我记录一下这个 case，后续得想办法避开

Reply:
这个问题不只有这一处，请检查现有的PR